installer/bundle/bundle_skel.sh

```diff
@@ -784,15 +767,19 @@ case "$installMode" in
         if [ $scx_installed -ne 0 -a $omi_installed -ne 0 ]; then
             echo "Installing OMS agent ..."
 
-            pkg_add_list $OMI_PKG omi
-            pkg_add_list $SCX_PKG scx
             pkg_add_list $OMS_PKG omsagent
 
             python_ctypes_installed
             if [ $? -eq 0 ]; then
                 pkg_add_list $DSC_PKG omsconfig
             fi
 
+            # Install SCX (and OMI)
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            SCX_STATUS=$?
+
             # Now actually install of the "queued" packages
             if [ -n "${add_list}" ]; then
                 if [ "$INSTALLER" = "DPKG" ]; then
```

Comment:
I'm fine with this. I just wanted you folks to give it consideration and decide what you wanted. I agree with your reasoning for now (without evidence to the contrary, why make the change). Once we have more solid data, we'll know (for sure) where to go.

In the case of SCX, we do have solid data (with at least one data point). Not so for OMS.